samples/python/evaluate_qrcode/evaluate_qrcode.py

```diff
@@ -0,0 +1,386 @@
+#!/usr/bin/env python
+
+'''
+
+This sample uses dataset from
+http://www.fit.vutbr.cz/research/groups/graph/pclines/pub_page.php?id=2012-SCCG-QRtiles
+and markup files located in markup/ folder
+for evaluation QR-code detection algorithm
+
+'''
+
+# Python 2/3 compatibility
+import optparse
+import os
+import sys
+from os.path import join
+
+import cv2
+import numpy as np
+import json
+
+
+def main():
```

Comment:
please change `def main():` to `if __name__ == "__main__":` cause its right form program's entry point.